master/cmd/determined-master/init.go

```diff
@@ -85,11 +85,6 @@ func registerConfig() {
 	registerString(flags, name("db", "ssl-root-cert"),
 		defaults.DB.SSLRootCert, "database ssl root cert path")
 
-	registerString(flags, name("scheduler", "fit"),
```

Comment:
this needs to be removed because our implementation of filling default values firstly uses the values using the option defaults. I want our scheduler and provisioner field values to go via the path of parsing `resource_manager` and `resource_pools`